lib/cosmos/tools/table_manager/table_item.rb

```diff
@@ -14,61 +14,42 @@ module Cosmos
 
   # Maintains knowledge of an item in a Table
   class TableItem < PacketItem
-    attr_reader :display_type
```

Comment:
I eliminated display_type and now just use the cosmos defaults. If you want to display hex then add a FORMAT_STRING "0x%0X"